BLSQ/iaso
# POLIO-1066 reasons for delay dropdown

POLIO-1037 was implemented with a limited choice of reasons for delay. We can now add a list provided by the client

Related JIRA tickets : POLIO-1066

## Self proofreading checklist


## Changes

- Update the text choices in the model + migration
- Update the dropdown list on the front-end + translations

## How to test

 Go to campaigns, try to edit round dates, you should see all the reasons for delay in the dropdaown (see Jira ticket for full list)

## Print screen / video

![Screenshot 2023-06-14 at 11 39 26](https://github.com/BLSQ/iaso/assets/38907762/a5c4a4e0-2b61-46e2-a356-8e9e950292f9)
